pkg/util/util_test.go

```diff
@@ -0,0 +1,38 @@
+package util
+
+import (
+	"testing"
+)
+
+func Test_getParentDirectory(t *testing.T) {
```

Comment:
Great you added a test! Ideally you would also test `CleanTargetPath()` (or its generic replacement).